### PR TITLE
WEB-321 - Creating new index page for tutorials

### PR DESCRIPTION
**Pull request by Santhosh-testsigma**
Creating a new index page for tutorials
image

### CODE REVIEW

**Comment by guganabu** [comment]
@Santhosh2022ts can you plz attach screenshot?

File: src/components/navcards.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+.card{
+  width: 48%;
+  border: 1px solid #DDDDDD;
+  border-radius: 2px;
+  @media (max-width: 425px) {
+    width: 100%;
+  }
+  &:hover{
+    background: #F2F3F3;
```
Comment (guganabu) [inline src/components/navcards.scss]: can we use variables and try to avoid using hard coded values?

File: src/components/navcards.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    font-size: 18px;
+    line-height: 45px;
+    color: #183247;
+    margin-bottom: 0 !important;
+  }
+  .card_desc{
+    font-weight: 400;
+    font-size: 15px;
+    color: #183247;
+    margin-top: 0 !important;
```
Comment (guganabu) [inline src/components/navcards.scss]: plz try to reduce !important usage

File: src/templates/page.scss [hunk 3]
```diff
@@ -528,4 +528,57 @@ code[class*="language-"] {
...
+  @media (max-width: 425px) {
+    margin-top: 1rem;
+  }
+}
+.homepage_footer{
+  flex-basis: 100%;
+  justify-content: space-between;
+  margin-bottom: 4rem;
+  .pagination_buttons{
+    @media (min-width: 768px) {
```
Comment (guganabu) [inline src/templates/page.scss]: plz use variables for defined screen sizes

File: src/templates/page.scss [hunk 4]
```diff
@@ -528,4 +528,57 @@ code[class*="language-"] {
...
+  }
+  .next_button{
+    width: fit-content;
+    @media (min-width: 768px) {
+      margin-top: 1rem;
+    }
+  }
+}
+.home-page_banner{
+  background-image: url("~/src/components/images/bg-banner.svg");
```
Comment (guganabu) [inline src/templates/page.scss]: why it's not inside assets?

Comment (Santhosh-testsigma) [reply src/templates/page.scss]: @guganabu  The docs and tutorials post content images are rendered from the AWS cloud and only very few images are used in components, that's why those images are saved inside of components. Also, only the Docs team has access to AWS.

Comment (guganabu) [reply src/templates/page.scss]: Can you move this inside src/images?
We don’t need to access from components

**Review by guganabu — APPROVED** [review]

**Pull request merged**